install-dependencies.sh

```diff
@@ -37,4 +37,4 @@ elif [[ $platform == "macosx" ]]; then
 fi
 
 sudo pip install --upgrade git+git://github.com/cloudpipe/cloudpickle.git@0d225a4695f1f65ae1cbb2e0bbc145e10167cce4  # We use the latest version of cloudpickle because it can serialize named tuples.
-sudo pip install --upgrade --verbose git+git://github.com/ray-project/numbuf.git@d1974afbab9f0f1bcf8af15a8c476d868ad31aff
```

Comment:
If you change this here, you should also change it in `install-on-macos.md` and `install-on-ubuntu.md`.